roles/openshift_node/tasks/upgrade/bootstrap_changes.yml

```diff
@@ -53,7 +33,7 @@
     owner: root
     group: root
     mode: 0600
-  with_items: "{{ system_kubeconfigs.files | default([]) | map(attribute='path') | list + [openshift.common.config_base+'/node/node.kubeconfig'] }}"
+  with_items: "{{ system_kubeconfigs.files | default([]) | map(attribute='path') | list }}"
```

Comment:
For scenarios where node.kubeconfig doesn't exist, like a rerun that fails when the node.kubeconfig has already been deleted.